scene.js

```diff
@@ -200,6 +200,29 @@ function createScene(options) {
       this.aspect[0] = aspectratio.x
       this.aspect[1] = aspectratio.y
       this.aspect[2] = aspectratio.z
+    },
+
+    setBounds: function(axisIndex, range) {
+      this.bounds[0][axisIndex] = range.min
+      this.bounds[1][axisIndex] = range.max
+    },
+
+    setClearColor: function(clearColor) {
+      this.clearColor = clearColor
+    },
+
+    clearRGBA: function() {
```

Comment:
Ok sure. I'm curious to know why you moved this logic in `gl-plot3d`?

Reply:
It includes a setter (for this.gl.clearColor parameter) and a low-level webgl clear call which used in both `gl-plot3d` as well as `Plotly/src/plots/gl3d/scene`.